app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+import app.errors
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine"):
+            expiration_date = visitor.get("vaccine").get("expiration_date")
+        else:
+            raise app.errors.NotVaccinatedError

```

Comment:
try to rewrite to increase readability

Reply:
Okay, I created new variables for calling dict